src/tooling/debugging/probe-rs.md

```diff
@@ -6,46 +6,126 @@ The [`probe-rs`][probe-rs] project is a set of tools to interact with embedded M
   - GDB support.
   - CLI for interactive debugging.
   - VS Code extension.
-- Real Time Transfer (RTT)
-  - Similar to app_trace component of IDF.
+- [Real Time Transfer (RTT)][rtt]
+  - Similar to [`app_trace` component of IDF][app-trace-idf].
 - Flashing algorithms
 
-More info about probe-rs & how to set up a project can be found on the [probe-rs] website.
+Follow the [installation][prober-rs-installation] and [setup][prober-rs-setup] instructions at the [`probe-rs`][probe-rs] website.
+
+Espressif products containing the [`USB-JTAG-SERIAL` peripheral][usb-jtag-serial] can use `probe-rs` without any external hardware.
 
 [probe-rs]: https://probe.rs/
 [openocd]: https://openocd.org/
 [pyocd]: https://pyocd.io/
 [segger-tools]: https://www.segger.com/
+[app-trace-idf]: https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/app_trace.html
+[rtt]: https://wiki.segger.com/RTT
+[prober-rs-installation]: https://probe.rs/docs/getting-started/installation/
+[prober-rs-setup]: https://probe.rs/docs/getting-started/probe-setup/
+[usb-jtag-serial]: index.md#usb-jtag-serial-peripheral
 
-## `USB-JTAG-SERIAL` Peripheral for ESP32-C3
-
-Starting from `probe-rs` v0.12, it is possible to flash and debug the ESP32-C3 with the built-in `USB-JTAG-SERIAL` peripheral, no need for any external hardware debugger. More info on configuring the interface can be found in the [official documentation][official-documentation].
-
-[official-documentation]: https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/api-guides/jtag-debugging/configure-builtin-jtag.html
+## Flashing with `probe-rs`
 
-## Support for Espressif Chips
+`probe-rs` can be used to flash your applications to your target, there are 2 supported image formats:
+- [ESP-IDF image format][idf-image]: This is the default format used by `espflash` and `cargo-espflash` and can be used in `probe-rs` using the `--format idf` argument
+  - Example command for flashing an ESP32-C3: `probe-rs run --chip esp32c3 --format idf`
+- `direct-boot`: Only supported in some Espressif products.
+  - Example command for flashing an ESP32-C3: `probe-rs run --chip esp32c3`
```

Comment:
We may just skip direct-boot as it may be removed

Reply:
Direct boot support has now been removed from `esp-hal`.

Reply:
Removed!